staging/src/k8s.io/api/networking/v1/generated.proto

```diff
@@ -404,7 +404,11 @@ message NetworkPolicyPort {
   optional k8s.io.apimachinery.pkg.util.intstr.IntOrString port = 2;
 }
 
-// NetworkPolicySpec provides the specification of a NetworkPolicy
+// NetworkPolicySpec provides the specification of a NetworkPolicy.  A suite of network
+// policies allow traffic ONLY if BOTH the egress AND the ingress are allowed.  For example,
+// if ingress is allowed by egress is not, or egress is allowed by ingress is not, the traffic
```

Comment:
same comment as earlier:

if ingress is allowed **but** egress is not, or egress is allowed **but** ingress is not

Reply:
omg yes!  thank you :)